front/app/utils/configs/participationMethodConfig/index.tsx

```diff
@@ -105,16 +105,7 @@ const ideationConfig: ParticipationMethodConfig = {
   supportsVotes: true,
   supportsComments: true,
   supportsTopicsCustomField: true,
-  onFormSubmission: ({ idea, phaseId }: FormSubmissionMethodProps) => {
-    if (idea) {
-      const urlParameters = `?new_idea_id=${idea.data.id}`;
-
-      clHistory.push({
-        pathname: `/ideas/${idea.data.attributes.slug}`,
-        search: urlParameters.concat(phaseId ? `&phase_id=${phaseId}` : ''),
-      });
-    }
-  },
+  onFormSubmission: () => {},
```

Comment:
Does `onFormSubmission` get set for any methods now that all forms are page based? Could be removed entirely otherwise. Assume it would now also be {} for voting or proposals too?